GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg

```diff
@@ -1,133 +1,347 @@
 PART
 {
-	name = ROC-LEMAscent
 	module = Part
+	name = ROC-LEMAscent
 	author = DECQ, Pap
-
+	
 	RSSROConfig = true
-
-	//	============================================================================
+	RP0conf = true
+	
+	//  ============================================================================
 	//	Model and Dimensions
-	//	============================================================================
-
+	//  ============================================================================
+	
 	MODEL
 	{
 		model = ROCapsules/Assets/DECQ/LEM/LEM_Ascent
+		scale = 1.0, 1.0, 1.0
+		rotation = 0, 180, 0
 	}
 
-	MODEL // Model for the Docking Port to Integrate it
-	{
-		model = ROCapsules/Assets/DECQ/LEM/LEM_dockingPort
-		scale = 1.175, 1.0, 1.175
-		position = 0.0, 1.325, 0.0
-	}
-
-	INTERNAL
-	{
-		name = ROC-LEMInternal
-	}
-
+    MODEL // Model for the Docking Port to Integrate it
+    {
+        model = ROCapsules/Assets/DECQ/LEM/LEM_dockingPort
+        scale = 1.175, 1.0, 1.175
+        position = 0.0, 1.325, 0.0
+        rotation = 0.0, 0.0, 0.0
+    }
+	
 	scale = 1.0
 	rescaleFactor = 1.0
-
+	
 	node_stack_top = 0.0, 1.405, 0.0, 0.0, 1.0, 0.0, 1
-	node_stack_bottom = 0.0, -1.375, 0.0, 0.0, -1.0, 0.0, 2
+    node_stack_bottom = 0.0, -1.375, 0.0, 0.0, -1.0, 0.0, 2
 
 	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
 	attachRules = 1,0,1,1,0
+	bulkheadProfiles = size1, size2
 
-	//	============================================================================
+    // --- FX definitions ---
+    fx_exhaustFlame_blue_small = 0.0, -0.5337813, 0.1355984, 0.0, 1.0, 0.0, running
+    fx_exhaustLight_blue = 0.0, -0.5337813, 0.1355984, 0.0, 0.0, 1.0, running
+		
+	//  ============================================================================
 	//	Title, Description, Category, Techs
-	//	============================================================================
-
-	TechRequired = unlockParts
-	cost = 150
-	entryCost = 1
+	//  ============================================================================
+	
+	title = Apollo LM Ascent Stage
+    manufacturer = Grumman Aerospace Corporation
+    description = The ascent stage and crew compartment of the Apollo Lunar Module. This has the ascent engine integrated.
+	
+	mass = 2.2
+	
 	category = Pods
 	subcategory = 0
-	title = Apollo LM Ascent Stage
-	manufacturer = Grumman Aerospace Corporation
-	description = The ascent stage and crew compartment of the Apollo Lunar Module. This has the ascent engine integrated.
+	
 	tags = apollo, moon, lunar, crew, armstrong, aldrin, cm, csm, saturn, v, lem, lm, module, ascent
-
+	
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	
```

Comment:
Why all the re-ordering?  Makes it look like a lot more changes that someone needs to review and see that, in fact, nothing changed.